inductiva/stellarators/scenarios/coils/coils.py

```diff
@@ -185,7 +196,35 @@ def from_curves_file(cls,
 
         return cls(coils, num_field_periods)
 
-    def simulate(self):
+    def simulate(
+        self,
+        simulator: Simulator = Simsopt(),
+        resource_pool_id: Optional[UUID] = None,
+        run_async: bool = False,
+        magnetic_field_filename: str = 'magnetic_field.npz',
+        plasma_surface_filename: str = 'input.QA',
+    ) -> tasks.Task:
+        """Simulates the scenario.
+
+        Args:
+            simulator: The simulator to use for the simulation.
+            resource_pool_id: The resource pool to use for the simulation.
+        """
+
+        task = super().simulate(
+            simulator,
+            resource_pool_id=resource_pool_id,
+            run_async=run_async,
+            magnetic_field_filename=magnetic_field_filename,
+            coil_coefficients_filename=COIL_COEFFICIENTS_FILENAME,
```

Comment:
I am struggling a bit with the naming of these variables. Their value seems to be simulator dependent, but they. have been declared as global variable for the scenario. Maybe this is common practice around scenarios but I would expected this to be named in a way that, at least, refers the corresponding simulator. E.g.

SIMOPT_DEFAULT_COIL_COEFFICIENTS_FILENAME

Ideally, these would belong to a structure or config file related with the simulator itself, but if we are including them in this file as global variable, there should be, at the very least, a mention to the simulator in their name.